packages/core/src/tools.ts

```diff
@@ -81,6 +81,17 @@ export namespace Tools {
 			height: parseFloat(el.style.height.replace("px", "") || el.offsetHeight)
 		};
 	}
+	/**
+	 * Returns an elements's x and y translations from its computed style
+	 * @param {HTMLElement} element
+	 * @returns an object containing the x and y translations
+	 */
+	export function getTranslationValues(elementRef: HTMLElement) {
```

Comment:
This still does something too specific. Take a look at this link:
https://stackoverflow.com/a/17838403

It'd be better if we can just parse the CSS transform values (or the inline transforms), and also return them in a more readable way:
```js
{
    translate: { x: 5, y: 10 },
    scale: { x: 1, y: 1.2 }
}
```

Reply:
Then you'd just take those results and do something like:
```js
result.translate.y
```

Reply:
yeah sure, I **replaced** a diff function in tools.ts that only returned the x translate coordinate in favour of **returning both** but I could add scale in case we need that later. i'll change the original parsing also

Reply:
after looking at this, it seems that returning a CSSStyleDeclaration object using HTMLElement.style **does not** return accurate (or any) transform that has been applied manually with d3. Only using window.getComputedStyle will take into account the applied transformations after the dom has been setup. They return the same CSSStyleDeclaration but when getting the transform information it is returned as a _matrix(a,b,c,d, tx, ty)_ rather than the nicer _"translate(5,6)rotate(5)"_ format to parse